app/javascript/components/import-image-form.jsx

```diff
@@ -0,0 +1,100 @@
+import React, { useMemo, useState } from "react";
+import PropTypes from "prop-types";
+import { connect } from "react-redux";
+import MiqFormRenderer from '@@ddf';
+import createSchema from './import-image-form.schema.js';
+
+const BLANK = {value: '-1', label: ''};
+
+const fetchProviders = (kind) => {
```

Comment:
Is it possible to move all these functions to components/import-image-form/helpers.js file?

Reply:
These contain API calls in them, so as discussed during the telco I'd leave them as ist only use string constants instead of directly encoding URLs